### ISSUE TITLE
npm i bem-join@latest fails

### ISSUE THREAD

**Issue by eballeste** [jednano/bem-join#28]
`npm i -S bem-join` fails:

node: 13.5.0
npm: 6.14.4

```
...
62 silly lifecycle bem-join@2.0.0~preinstall: Returned: code: 1  signal: null
63 info lifecycle bem-join@2.0.0~preinstall: Failed to exec preinstall script
64 timing action:preinstall Completed in 3698ms
...
75 verbose stack Error: bem-join@2.0.0 preinstall: `npx npm-force-resolutions`
75 verbose stack Exit status 1
75 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
75 verbose stack     at EventEmitter.emit (events.js:305:20)
75 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
75 verbose stack     at ChildProcess.emit (events.js:305:20)
75 verbose stack     at maybeClose (internal/child_process.js:1028:16)
75 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
76 verbose pkgid bem-join@2.0.0
77 verbose cwd /Users/divisionofdev/Documents/sites/test
78 verbose Darwin 18.7.0
79 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "i" "-S" "bem-join"
80 verbose node v13.5.0
81 verbose npm  v6.14.4
82 error code ELIFECYCLE
83 error errno 1
84 error bem-join@2.0.0 preinstall: `npx npm-force-resolutions`
84 error Exit status 1
85 error Failed at the bem-join@2.0.0 preinstall script.
85 error This is probably not a problem with npm. There is likely additional logging output above.
86 verbose exit [ 1, true ]
```

**Comment by eballeste** [comment]
also showing missing package-lock.json files before it dies even though it exists:
```
ENOENT: no such file or directory, open './package-lock.json'
```

**Comment by eballeste** [comment]
running: `npx npm-force-resolutions` by itself results in:
```
npx: installed 5 in 2.378s
```

**Comment by eballeste** [comment]
`1.2.1` works fine

**Comment by jednano** [comment]
Thanks for pointing this out. I'll have a look when I have some time.

**Comment by jednano** [comment]
Fixed in v2.0.2.
